### PR TITLE
[mql mutation]update mql sever graphql schema for `updateMqlServerConfig`

### PR DESCRIPTION
**Pull request by conglei**
# Changes
This is to update the mql schema per change in the PR https://github.com/transform-data/hedgehog/pull/1298 and it also bump update the core backend schema to the lasted version.

# Security Implications
No

### CODE REVIEW

**Comment by conglei** [comment]
I don't know `UpdateMqlServerConfigMutationVariables` is not updated..

@jjrajani is there anything I did wrong?

```ts
export type UpdateMqlServerConfigMutationVariables = Exact<{
  mqlServerId: Scalars['Int'];
  dwPassword: Scalars['String'];
  tfdApiKey: Scalars['String'];
}>;
```

Thanks!

**Review by jjrajani — CHANGES_REQUESTED** [review]
The package version in `package.json` will also need to be updated.

File: schemas/core/core.graphql [hunk 1]
```diff
@@ -188,11 +185,9 @@ type Team {
   updatedAt: DateTime
   deactivatedAt: DateTime
   description: String
-  createdTs: Int
-  deactivatedTs: Int
   createdByUser: User
   organization: Organization
-  members(pageNumber: Int, pageSize: Int, orderBy: TeamMemberUserOrderBy, desc: Boolean): [TeamMember]
+  members(pageNumber: Int, pageSize: Int, orderBy: UserUserOrderBy, desc: Boolean): [TeamMember]
```
Comment (courtneyholcomb) [inline schemas/core/core.graphql:190]: I think I may have introduced a but when generating the typename for this orderBy - it shouldn't have changed. (It gets generated in code, not hardcoded, hence me not catching this sooner)

**Comment by conglei** [comment]
Just updated the schema again against production graphql endpoint. Still I'm very confused on why `UpdateMqlServerConfigMutationVariables` is not updated.. supposedly, the new added field `modeCreds` should be shown up there.

**Review by jjrajani — APPROVED** [review]

**Pull request merged**